src/pages/page-home/index.js

```diff
@@ -47,6 +48,7 @@ class Page extends PageMixin(ElementLiteLit(HTMLElement, style.toString())) {
   }
 
   get about () {
+    updateState('currentProgress', 'loaded');
```

Comment:
what is this for?

Can you show pictures (i'm just lazy running it right now)

Reply:
@jkga

Reply:
That is for hiding the progress bar when page is loaded. It is inspired by [gdgph's](https://www.gdgph.org/#/) progress bar .You can [view it here](https://searca-my.sharepoint.com/:i:/g/personal/jkga_searca_org/EURMSvPPncVAlEUYu4tUDUMBW2Pwcz3IPyqvMvlt3rRX2g?e=cE8HO2)